project1-sps/script.js

```diff
@@ -1,4 +1,144 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// initialise variables
+let boolUserEnteredName = 0;
```

Comment:
Good application of a boolean function here. But could we make our code more efficient by omitting this variable? Maybe a test if the username string is empty i.e. `if (userName === "")` will serve the same purpose!

Reply:
Sure. Maybe if I added some variable to assign/update and keep track the "mode of the game" in the code, we can evaluate  the 'userInput' into the main function directly within the code blocks for each game mode?